src/tools/SwitchingFunction.cpp

```diff
@@ -305,6 +308,9 @@ double SwitchingFunction::calculate(double distance,double&dfunc)const{
       double tmp1=std::tanh(rdist);
       result = 1.0 - tmp1;
       dfunc=-(1-tmp1*tmp1);
+    }else if(type==none){
+      result = rdist * rdist;
+      dfunc = rdist;
```

Comment:
Also, this won't behave "like DRMSD" whenever distance is less than the reference:
rdist=0 as well as rdist<0. return 1. immediately. instead of desired zero and a (negative side) parabola, respectively.
Also, if(distance>dmax)  - 0.0 is returned rather than high value of DRMSD component.


Would be very useful if one could to that though...